tests/Fixtures/DummyTranslatable.php

```diff
@@ -19,6 +17,6 @@ class DummyTranslatable extends AbstractTranslatable
      */
     protected function createTranslation(): TranslationInterface
     {
-        return new DummyTranslation();
+        return new DummyTranslation('en', null);
```

Comment:
@SpartakusMd 'en' is already fallback locale and it should be set if no locale is provided.

Reply:
Removed the parameters as they are not needed.